pkg/webui/components/file-input/file-input.styl

```diff
@@ -0,0 +1,61 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@require '../button/button.styl'
```

Comment:
can we avoid doing this?

Reply:
why?

Reply:
it breaks the encapsulation of components. we should prefer composing react components, rather than just importing stuff we need. Is there a way to reuse button as a component?

Reply:
My workaround was build upon changing the styling of the `<label>` element, to be able to use its native click event. I changed that to use our `<Button />` component directly and trigger the click event programmatically, which should be fine as well and saves us the styling import.